package.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
once the code is merged we should delete the existing package and release it again containing the readme

Reply:
This is something where I haven't had time to think deeply about how best to structure things. If we deploy like this the README is actually the README of the `tooling` repo which is not what we want.

An option I have in mind is to have a subfolder, ' mento-core-ts' with the `package.json` and the deployment scripting and everything that has to do with the NPM package, and not have that at the root of the repo. But I can't tell how ugly that is.

Otherwise, we can consider actually not going down the `tooling` route as we initially thought, but have two separate repositories: 

- `mento-protocol/mento-deployment`
- `mento-protoco/mento-core-ts` 

Which both depend on `mento-protocol/mento-core`, and potentially `mento-protocol/mento-core-ts` might depend on `mento-protocol/mento-deployment` to access addresses (when we move away from the registry pattern).

I'm starting to think this might be a bit cleaner. Otherwise we're kinda doing a half monorepo, half not approach.

Thoughts @mento-protocol/core

Reply:
@bowd @ninocomputer Resolving this now. This comment is no longer applicable to this repo. We have [a new issue](https://github.com/mento-protocol/mento-core-ts/issues/1) to create the `mento-core-ts` repo and can add the correct readme at the root of that repo.